inst/extdata/qualification-workflow-template.R

```diff
@@ -94,6 +114,12 @@ createQualificationReport <- function(qualificationRunnerFolder,
   }
 
   #-------- STEP 3: Run Qualification Workflow  --------#
+  # If version info is provided update title page
+  if(!is.null(versionInfo)){
+    titlePageFile <- file.path(reInputFolder, "Content/titlepage.md") 
+    adjustTitlePage(titlePageFile, qualificationVersionInfo = versionInfo)
```

Comment:
Let's check if the title page file exists, otherwise adjustTitlePage would crash